services/venice-router/src/test/java/com/linkedin/venice/router/RouteHttpRequestStatsTest.java

```diff
@@ -20,20 +31,30 @@ public class RouteHttpRequestStatsTest {
   private RouteHttpRequestStats stats;
   private RouterHttpRequestStats routerHttpRequestStats;
 
+  private VeniceMetricsRepository localMetricRepo;
+
   @BeforeSuite
   public void setUp() {
     MetricsRepository metrics = MetricsRepositoryUtils.createSingleThreadedVeniceMetricsRepository();
     reporter = new MockTehutiReporter();
     metrics.addReporter(reporter);
-
+    MetricConfig metricConfig = new MetricConfig().timeWindow(1, TimeUnit.SECONDS);
```

Comment:
To avoid the duplicate code, we can use `InFlightRequestStat` to construct the in-flight-sensor here, right?

Reply:
Sure